### PR TITLE
Fixed Phaser.Actions.Spread ignoring the min parameter

### PR DESCRIPTION
**Pull request by galman33**
This PR (delete as applicable)

* Fixes a bug

Describe the changes below:
 
* Fixed Phaser.Actions.Spread ignoring the min parameter

### CODE REVIEW

File: src/actions/Spread.js [hunk 1]
```diff
@@ -40,14 +40,14 @@ var Spread = function (items, property, min, max, inc)
     {
         for (i = 0; i < items.length; i++)
         {
-            items[i][property] += i * step;
+            items[i][property] += i * step + min;
```
Comment (jorbascrumps) [inline src/actions/Spread.js]: Should `min` be initialized with a default value (perhaps 0?) to prevent `NaN` result when `min` is omitted?

Comment (galman33) [reply src/actions/Spread.js]: I believe min isn't an optional parameter (?)

Comment (jorbascrumps) [reply src/actions/Spread.js]: My apologies! You are correct.

**Comment by photonstorm** [comment]
👍

**Pull request merged**